Procfile

```diff
@@ -1 +1 @@
-worker: node src/spawn_tests.js
+worker: npm run compile && node lib/main.js test
```

Comment:
Probably don't want to add this `npm run compile` step here. Any reason you did? We can probably find another way to achieve the same purpose.